pkg/image/apis/image/helper.go

```diff
@@ -41,6 +46,16 @@ const (
 	ImportRegistryNotAllowed = "registry is not allowed for import"
 )
 
+// knownSignatureStores is a list of known signature stores. The key represents
+// a registry name and it maps to a signature store for that registry. The
+// signature store has to be HTTPS and it must offer image signatures based on
+// this format: https://STORE_URL/IMAGE_NAME/signature-1
+// The response must be valid image signature blob.
+// TODO: Maybe this should be configurable via master config?
```

Comment:
@mtrmac the server needs to know from where it should fetch the signature from when the image is being imported into openshift (and pushed to integrated registry). We don't rely on docker when fetching image metadata.